AutomatorServer/android/app/src/androidTest/java/pl/leancode/automatorserver/PatrolAutomator.kt

```diff
@@ -198,7 +187,7 @@ class PatrolAutomator {
         pressBack() // Hide keyboard.
     }
 
-    fun swipe(swipe: SwipeCommand) {
+    fun swipe(swipe: Contracts.SwipeCommand) {
```

Comment:
I hate this prefix but haven't found a way to get rid of this. Same on Swift side.